### DISCUSSION TITLE
How to find the last Linux-i386.AppImage

### DISCUSSION THREAD

**Discussion by jasteele12** [standardnotes/forum#3538] [General]
I have standard-notes-3.23.237-Linux-i386.AppImage on an Acer One running Zorin OS Lite 15.3 32-bit. I picked that SN version when I got tired (and almost carpal injury) from scrolling through page after page under Releases.

Of course the check for update fails with a path moved message. How do I find out the last 32-bit release without losing a limb?
A search for i386 was no help. Even a ballpark release page number could be useful. ;-P

Thanks for any help!

**Comment by effieeee** [comment]
Unfortunately the team no longer packages a 32-bit AppImage in the latest versions of the app 😢 The link below is the last release where a 32-bit AppImage was included:

https://github.com/standardnotes/app/releases/tag/%40standardnotes%2Fdesktop%403.130.5
